pages/common/hugo.md

```diff
@@ -0,0 +1,31 @@
+# hugo
+
+> Hugo is a fast and flexible static site generator.
+
+- Create a new Hugo site:
+
+`hugo new site {{path/to/site}}`
+
+- Create a new Hugo theme (themes may also be downloaded from https://themes.gohugo.io/):
```

Comment:
The theme gets added to an existing site. In fact, `hugo` will not allow you to create a new theme unless you are within a site's directory which is why I placed this example after the "creating a new site" example.